packages/frontend/src/components/DashboardTabs/index.tsx

```diff
@@ -240,38 +240,65 @@ const DashboardTabs: FC<DashboardTabsProps> = ({
                                               backgroundColor: 'white',
                                               flexGrow: 1,
                                           },
+                                          tabsList: {
+                                              flexWrap: 'nowrap',
+                                          },
+                                          tab: {
+                                              borderBottom:
+                                                  '1px solid var(--mantine-color-gray-3)',
+                                          },
                                       }
                                     : undefined
                             }
                             variant="outline"
                         >
                             {sortedTabs && sortedTabs?.length > 0 && (
                                 <Tabs.List bg="gray.0" px="lg">
-                                    {sortedTabs?.map((tab, idx) => {
-                                        return (
-                                            <DraggableTab
-                                                key={tab.uuid}
-                                                idx={idx}
-                                                tab={tab}
-                                                isEditMode={isEditMode}
-                                                sortedTabs={sortedTabs}
-                                                currentTabHasTiles={
-                                                    currentTabHasTiles
-                                                }
-                                                isActive={
-                                                    activeTab?.uuid === tab.uuid
-                                                }
-                                                setEditingTab={setEditingTab}
-                                                handleDeleteTab={
-                                                    handleDeleteTab
-                                                }
-                                                setDeletingTab={setDeletingTab}
-                                            />
-                                        );
-                                    })}
+                                    <ScrollArea
+                                        type="hover"
+                                        offsetScrollbars
+                                        scrollHideDelay={200}
+                                        variant="primary"
+                                        scrollbarSize={6}
+                                        styles={{
+                                            viewport: {
+                                                paddingBottom: 0,
```

Comment:
removed padding bottom so there's no extra space and used variant primary from our theme

Reply:
Thank you, I'm trying to learn how to use Mantine better but I need more practice.